modules/preview-react/_examples/stories/examples/SidePanelWithOverlay.tsx

```diff
@@ -22,7 +17,9 @@ export const Basic = () => {
     <Flex height={320} backgroundColor="soap100" position="relative">
       <StyledPanel touched={true} variant="alternate">
         <Flex alignItems="center" paddingY="s" paddingX="s">
-          <StyledHeader {...labelProps}>Alternate Panel</StyledHeader>
+          <TypeBodyLevel as="h3" size="large" fontWeight="bold" color="licorice500">
+            Alternate Panel
+          </TypeBodyLevel>
```

Comment:
for `h` tags to we want to use `TypeBodyHeading` ?

Reply:
I notice that it's repeating in different components. Based on code this heading has text styles prop as a body text but bold. So i think this inconsistence should be reviewed deeply.

Reply:
fair, maybe a question for the team?

Reply:
Yes, I want to check design specs first and maybe involve design team for this discussion.